starter-code/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "chucknorris-io": "^1.0.4",
     "ejs": "^2.5.5",
-    "express": "^4.14.0"
+    "express": "^4.14.0",
+    "express-ejs-layouts": "^2.3.1"
   }
 }
```

Comment:
Remember to add the scripts that you are going to use, now you only have the "test" one (line 7). E.g:
```
"scripts": {
    "start": "node app.js",
    "start-dev": "nodemon --inspect app.js"
}
```